aslam_cv_cameras/include/aslam/cameras/undistorter-mapped-inl.h

```diff
@@ -88,9 +87,16 @@ inline std::unique_ptr<MappedUndistorter> createMappedUndistorter(
   common::buildUndistortMap(
       *input_camera, *output_camera, CV_16SC2, map_u, map_v);
 
-  return std::unique_ptr<MappedUndistorter>(new MappedUndistorter(
-      input_camera, output_camera, map_u, map_v, interpolation_type));
+  // Convert map to non-fixed point representation for easy lookup of values.
+  cv::Mat map_u_float = map_u.clone();
```

Comment:
does it make sense to clone the cv::Mat and then later I guess you just overwrite everything again or even reallocate them? Why not initialize them to the right size and type here or inside the function?